### PR TITLE
newsletter 2019-11-01

### PR DESCRIPTION
**Pull request by ryan-shea**
woah!!!! november!!!

### CODE REVIEW

File: app/retail/emails.py [hunk 1]
```diff
@@ -898,8 +898,8 @@ def render_start_work_applicant_expired(interest, bounty):
 def render_new_bounty_roundup(to_email):
     from dashboard.models import Bounty
     from django.conf import settings
-    subject = "❗ A New Quest"
-    new_kudos_pks = [5065, 1921, 73]
+    subject = "Updates, Updates, Updates"
```
Comment (owocki) [inline app/retail/emails.py]: 'Review of Gitcoin Quadratic Funding Round 3'

Comment (owocki) [reply app/retail/emails.py]: New Subject:
 The Quadratic Freelancer

File: app/retail/emails.py [hunk 2]
```diff
@@ -920,13 +920,13 @@ def render_new_bounty_roundup(to_email):
...
 </p>
 <p>
-    ⏰ It’s almost that time. Web3 World is around the corner, and we’re stoked to offer another tremendous invitation for learning, earning, and of course, hacking. Watch out for a tweet later today with an official announcement post. Not following us on Twitter yet? <a href="https://twitter.com/gitcoin">Let’s fix that.</a>
+    🚨Kudos Alert!🚨 We've released a brand new feature with Kudos – the ability to mint your own. Like your favorite cappuccino machine, it lets you make your own delicious <strike>coffee</strike> Kudos all by yourself. If you have Kudos you want to put on the marketplace, all you've have to do is fill out a form, add your artwork, and watch as it makes its way to the Gitcoin Kudos marketplace! How neat is that. Check out the self serve feature <a href="https://gitcoin.co/kudos/new">here.</a>
```
Comment (owocki) [inline app/retail/emails.py]: 🚨New Kudos Feature Alert!🚨 We've released a brand new feature for Kudos – the ability to mint your own *without asking us to help*. Like your favorite cappuccino machine, it lets you make your own delicious <strike>coffee</strike> Kudos all by yourself. If you have Kudos you want to put on the marketplace, all you've have to do is fill out a form, add your artwork, and watch as it makes its way to the Gitcoin Kudos marketplace! How neat is that. Check out the self serve feature

File: app/retail/emails.py [hunk 3]
```diff
@@ -935,7 +935,7 @@ def render_new_bounty_roundup(to_email):
         The Gitcoin Livestream is on for this week! Join us <a href="https://gitcoin.co/livestream"> at 2PM ET this Friday!</a>
```
Comment (owocki) [inline app/retail/emails.py]: Kevin Owocki will be presenting Gitcoin Quests

File: app/retail/emails.py [hunk 4]
```diff
@@ -920,13 +920,13 @@ def render_new_bounty_roundup(to_email):
 Hey Gitcoiners,
 </p>
 <p>
-    We’re proud to announce that this week marks the launch of our newest product, Gitcoin Quests. Quests is an enjoyable, gamified way to learn about the Web3 ecosystem, compete with your friends, earn rewards, and level up your knowledge. We’ve been working hard for the past couple months and are excited to see how you like it. <a href="https://gitcoin.co/blog/gitcoin-quests-is-now-live/">To begin your adventure, click here.</a>
+    🌱 Gitcoin Grants CLR Round 3 was a massive, massive success. With wide ranging support from across the entire Ethereum ecosystem, we were blown away with the support you provided: 1,982 unique contributions to 68 projects worth $170,000. If you'd like to take a closer look at where the record $270k ended up, take a look at our blog post. <a href="https://gitcoin.co/blog/gitcoins-q3-match/">It's living here.</a>
```
Comment (owocki) [inline app/retail/emails.py]: can we pls link to this post at some point?  https://vitalik.ca/general/2019/10/24/gitcoin.html

the number one person in the entire ecosystem wrote about us.. and it's a damn good article. we should let people know to check it out.

New Copy: 

'''

Gitcoin Grants CLR Round 3 was a massive, massive success. With wide ranging support from across the entire Ethereum ecosystem, we were blown away with the support you provided: 1,982 unique contributions to 68 projects worth $170,000. In total, between individual contributions the round ended up worth $270k.
</p>
<p>
In a review of Gitcoin Grants Round 3, Ethereum Co-Founder Vitalik Buterin says "If Gitcoin Grants does continue operating regularly, and attracts and expands its pool of donations, we could be very close to seeing the first "quadratic freelancer" - someone directly "working for the public", funded by donations boosted by quadratic matching subsidies."<a href=" https://vitalik.ca/general/2019/10/24/gitcoin.html">Click here to read the article.</a>
'''

Comment (ryan-shea) [reply app/retail/emails.py]: didn't see this! so cool.

**Comment by ryan-shea** [comment]
@owocki changes updated

**Pull request merged**